test/src/module/command/restoreTest.c

```diff
@@ -2474,15 +2521,15 @@ testRun(void)
             "P00 DETAIL: remove special file '" TEST_PATH "/pg/pipe'\n"
             "P00 DETAIL: create symlink '" TEST_PATH "/pg/pg_tblspc/1' to '" TEST_PATH "/ts/1'\n"
             "P00 DETAIL: create path '" TEST_PATH "/pg/pg_tblspc/1/16384'\n"
-            "P01 DETAIL: restore file " TEST_PATH "/pg/postgresql.auto.conf (15B, 44.12%)"
+            "P01 DETAIL: restore file " TEST_PATH "/pg/postgresql.auto.conf (15B, 44.11%)"
             " checksum 37a0c84d42c3ec3d08c311cec2cef2a7ab55a7c3\n"
-            "P01 DETAIL: restore file " TEST_PATH "/pg/postgresql.conf (10B, 73.53%) checksum"
+            "P01 DETAIL: restore file " TEST_PATH "/pg/postgresql.conf (10B, 73.52%) checksum"
             " 1a49a3c2240449fee1422e4afcf44d5b96378511\n"
             "P01 DETAIL: restore file " TEST_PATH "/pg/PG_VERSION - exists and matches size 4 and modification time 1482182860"
             " (4B, 85.29%) checksum 8dbabb96e032b8d9f1993c0e4b9141e71ade01a1\n"
-            "P01 DETAIL: restore file " TEST_PATH "/pg/size-mismatch (1B, 88.24%) checksum"
+            "P01 DETAIL: restore file " TEST_PATH "/pg/size-mismatch (1B, 88.23%) checksum"
             " c032adc1ff629c9b66f22749ad667e6beadf144b\n"
-            "P01 DETAIL: restore file " TEST_PATH "/pg/tablespace_map (0B, 88.24%)\n"
+            "P01 DETAIL: restore file " TEST_PATH "/pg/tablespace_map (0B, 88.23%)\n"
```

Comment:
Why has this changed?

Reply:
Since I [changed](https://github.com/arenadata/pgbackrest/pull/55/files#diff-9cfdda13c7fe664728b8fb1c44176c0408e96ac403d1e15e2f96265e5ddfa35fR2269) from double to int. Similar as it [implemented](https://github.com/arenadata/pgbackrest/blob/9546b9c5d0e4da7bd952b01cc3374c26e9f90ea0/src/command/backup/backup.c#L1466) inside backupJobResult.